Microsoft/WebTemplateStudio
# side bar design change + copy change for entire wizard

Home page:
- changed "Welcome to WTS " to "create your web  app in seconds"

frameworks & azure:
- changed details to learn more

pages:
- changed details to preview

Azure Services Page:
- changed styling of modal + made it more responsive 

Right side bar:
- changed styling
- added icons for hide, edit, and services 
- replace send feedback button with report an issue link

How to test:
- check the text
- it should look like this when you build it:
![forms](https://user-images.githubusercontent.com/31115095/61341726-69321d80-a7fc-11e9-8c0b-04893bc837b6.PNG)
![rightsidebar](https://user-images.githubusercontent.com/31115095/61341739-7b13c080-a7fc-11e9-90d0-241a22ea5a39.PNG)
